src/main/java/com/google/devtools/build/lib/rules/proto/ProtoConfiguration.java

```diff
@@ -199,11 +199,26 @@
                 + "been superseded by the strip_import_prefix= and import_prefix= attributes")
     public boolean disableProtoSourceRoot;
 
+    @Option(
+        name = "incompatible_load_proto_rules_from_bzl",
+        defaultValue = "false",
+        documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
+        effectTags = {OptionEffectTag.LOADING_AND_ANALYSIS},
+        metadataTags = {
+          OptionMetadataTag.INCOMPATIBLE_CHANGE,
+          OptionMetadataTag.TRIGGERED_BY_ALL_INCOMPATIBLE_CHANGES
+        },
+        help =
+            "If enabled, direct usage of the native Protobuf rules is disabled. Please use "
+                + "the Starlark rules instead https://github.com/bazelbuild/rules_proto")
+    public boolean loadProtoRulesFromBzl;
+
     @Override
     public FragmentOptions getHost() {
       Options host = (Options) super.getHost();
       host.disableLegacyProvider = disableLegacyProvider;
       host.disableProtoSourceRoot = disableProtoSourceRoot;
+      host.loadProtoRulesFromBzl = loadProtoRulesFromBzl;
```

Comment:
Nice! I am especially sensitive to this because I missed a line similar to this in https://github.com/bazelbuild/bazel/commit/ce65de4ee8e006137c8d927fa474384c0d940cb2#diff-f5054593a10aa398dc6a4da2eb4bf7e2 :)